cmd/agent/daemon/daemon.go

```diff
@@ -75,8 +89,14 @@ func NewCommand(version string) *cobra.Command {
 				os.Exit(1)
 			}
 
+			apiKey, err := lookupConfigVariable("API_KEY")
```

Comment:
Would be nice to also add the same validation for  `CASTAI_CLUSTER_ID` and `CASTAI_API_GRPC_ADDR`

Reply:
Validation has been added